workers/unity/Packages/com.improbable.gdk.tools/.DownloadCoreSdk/Program.cs

```diff
@@ -0,0 +1,133 @@
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            if (args.Length == 0)
+            {
+                Console.Error.WriteLine("Specify the version of the CoreSdk to download.");
+                Environment.Exit(1);
+            }
+
+            var coreSdkVersion = args[0];
+            var nativeDependenciesPath = Path.GetFullPath("./Assets/Plugins/Improbable/Core");
+            var managedDependenciesPath = Path.GetFullPath("./Assets/Plugins/Improbable/Sdk");
+            var tempPath = Path.GetFullPath($"./build/CoreSdk/{coreSdkVersion}");
+            var projectPath = Path.GetFullPath("../../");
+
+            var Packages = new List<Package>
+            {
+                new Package
+                {
+                    Type = "worker_sdk",
+                    Name = "c-dynamic-x86_64-msvc_mt-win32",
+                    TargetPath = $"{nativeDependenciesPath}/Windows/x86_64",
+                    CleanPaths = new List<string> {"include", "worker.lib"}
+                },
+                new Package
+                {
+                    Type = "worker_sdk",
+                    Name = "c-dynamic-x86_64-gcc_libstdcpp-linux",
+                    TargetPath = $"{nativeDependenciesPath}/Linux/x86_64",
+                    CleanPaths = new List<string> {"include"}
+                },
+                new Package
+                {
+                    Type = "worker_sdk",
+                    Name = "c-bundle-x86_64-clang_libcpp-macos",
+                    TargetPath = $"{nativeDependenciesPath}/OSX",
+                    CleanPaths = new List<string> {"include"}
+                },
+                new Package
+                {
+                    Type = "worker_sdk",
+                    Name = "csharp_core",
+                    TargetPath = $"{managedDependenciesPath}"
+                },
+                new Package
+                {
+                    Type = "schema",
+                    Name = "standard_library",
+                    TargetPath = $"{projectPath}/build/dependencies/schema/standard_library"
+                },
+                new Package
+                {
+                    Type = "tools",
+                    Name = "schema_compiler-x86_64-win32",
+                    TargetPath = $"{tempPath}/schema_compiler",
+                    Platform = OSPlatform.Windows
+                },
+                new Package
+                {
+                    Type = "tools",
+                    Name = "schema_compiler-x86_64-macos",
+                    TargetPath = $"{tempPath}/schema_compiler",
+                    Platform = OSPlatform.OSX
+                }
+            };
+
+            try
+            {
+                if (!Directory.Exists(tempPath)) Directory.CreateDirectory(tempPath);
+
+                foreach (var package in Packages.Where(p =>
+                    !p.Platform.HasValue || RuntimeInformation.IsOSPlatform(p.Platform.Value)))
```

Comment:
Perhaps add a `IsPlatformAgnostic` property to `Package` that returns `!Platform.hasValue` 

Will make this code easier to read